src/Workspaces/CoreTestUtilities/ExportProviderCache.cs

```diff
@@ -163,6 +163,9 @@ bool IsExcludedPart(ComposablePartDefinition part)
             }
         }
 
+        public static ComposableCatalog WithoutPartsOfTypes(this ComposableCatalog catalog, params Type[] types)
```

Comment:
>WithoutPartsOfTypes [](start = 40, length = 19)

I couldn't figure out if/where this method is called. Can you confirm it's used?

Reply:
It is not.  Sorry, it was a different approach that didn't pan out.